apps/dev-app/server.js

```
@@ -0,0 +1,150 @@
+
+  ws.on('message', (data) => {
+    try {
+      const message = JSON.parse(data.toString());
+      console.log('📨 Received message:', message);
+
+      const clientSubs = clientSubscriptions.get(ws);
+
+      switch (message.type) {
+        case 'subscription':
+          const subKey = JSON.stringify(message.payload.key);
+          clientSubs.add(subKey);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)